### DISCUSSION TITLE
Blocked for asking a question - Quasar

### DISCUSSION THREAD

**Discussion by Cyberwebnerd** [community/community#77890] [New to GitHub]
I was in the Quasar organisation page, and I had a question as I am new to this sort of thing.   My question was asking for help from other members for what to do next.

I get an email saying that I am blocked from the Quasar Organisation page as, apparently asking for help, is against community guidelines.  How can i reach out to someone there for help.  I feel like it discourages those who are new from wanting to ask for help

**Comment by queenofcorgis** [comment]
Hi there @techscamhunter, 

Organizations and projects hosted on GitHub are separate from the GitHub community and blocking members is up to the maintainers of that software. Most methods of contacts would be found within in the repository for the project/organization. 

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place to assist you.

**Comment by NatoBoram** [comment]
* https://github.com/quasar/Quasar/issues/1224

It looks like your question was completely unrelated to Quasar.

Not only that, but you were asking for help on how to send a rat via email. I'm sure that Quasar's maintainers have other things to do than to teach you how to use a computer.
